zabapgit.prog.abap

```diff
@@ -13307,6 +13322,42 @@ CLASS lcl_objects IMPLEMENTATION.
 
     lcl_objects_activation=>activate( ).
 
+
+    LOOP AT lt_late_deser_results ASSIGNING <ls_result>.
```

Comment:
can we move this loop to a new method, which imports lt_late_deser_results? also it looks similar to the LOOP AT lt_results above, so perhaps that loop can also be replaced with the new method call